### ISSUE TITLE
Capsule networks for low-data transfer learning

### ISSUE THREAD

**Issue by Swall0w** [Swall0w/papers#380]
Gritsevskiy, Andrew, Korablyov, Maksym

We propose a capsule network-based architecture for generalizing learning to
new data with few examples. Using both generative and non-generative capsule
networks with intermediate routing, we are able to generalize to new
information over 25 times faster than a similar convolutional neural network.
We train the networks on the multiMNIST dataset lacking one digit. After the
networks reach their maximum accuracy, we inject 1-100 examples of the missing
digit into the training set, and measure the number of batches needed to return
to a comparable level of accuracy. We then discuss the improvement in low-data
transfer learning that capsule networks bring, and propose future directions
for capsule research.

https://arxiv.org/abs/1804.10172
